reactos/drivers/usb/usbhub_new/pnp.c

```diff
@@ -562,7 +562,7 @@ USBH_FdoCleanup(IN PUSBHUB_FDO_EXTENSION HubExtension)
                 PortExtension->PortPdoFlags &= ~USBHUB_PDO_FLAG_WAIT_WAKE;
                 PortExtension->PdoWaitWakeIrp = NULL;
 
-                if (PortIdleIrp->Cancel ||
+                if (PortWakeIrp->Cancel ||
                     !IoSetCancelRoutine(PortWakeIrp, NULL))
                 {
                     if (!InterlockedDecrement(&HubExtension->PendingRequestCount))
```

Comment:
pnp.c:568: So we can get here because the CancelRoutine is already executing. Should we really still complete the IRP below? Should this case set PortWakeIrp to NULL maybe?